lib/cli.js

```diff
@@ -223,7 +223,7 @@ export default async function loadCli() { // eslint-disable-line complexity
 
 	const combined = {...conf};
 
-	for (const flag of Object.keys(FLAGS)) {
+	for (const flag of ['color', ...Object.keys(FLAGS)]) {
```

Comment:
Why not add to `FLAGS`?

Reply:
Because `FLAGS` is used to define options for subcommands, whereas `color` applies at top-level: https://github.com/avajs/ava/blob/392ab65cd4d203643e5bb29aa7f152056fb6040b/lib/cli.js#L162-L183